components/Toc.vue

```diff
@@ -142,7 +142,7 @@ export default {
   max-height 100vh
   max-width 220px
   overflow-y auto
-  padding-top $navbarHeight
```

Comment:
just want to be sure that's a good idea to remove some vars ? Easy to override ?